nipype/interfaces/ants/registration.py

```diff
@@ -322,7 +322,7 @@ class RegistrationInputSpec(ANTSCommandInputSpec):
         low=0.0, high=1.0, value=0.0, argstr='%s', usedefault=True, desc="The Lower quantile to clip image ranges")
     collapse_linear_transforms_to_fixed_image_header = traits.Bool(
         argstr='%s', default=False, usedefault=True, desc='')
-
+    float_computations = traits.Int(argstr='--float %d', value=0, desc="Use single floating point for computations")
```

Comment:
After a quick look at the ANTs command line parser implementation it seems that the parser adds a default '1' if the argument following the command line option string starts with '-' and can not interpreted as a float (i.e. it is supposedly the start of the next command line option string), or if there are no arguments left.  
However, one could also add "--float 0" to the command line ...